spec/functions_spec.rb

```diff
@@ -29,17 +29,18 @@
       :asin, :acos, :atan, :acsc, :asec, :acot,
       :sinh, :cosh, :tanh, :csch, :sech, :coth,
       :asinh, :acosh, :atanh, :acsch, :asech, :acoth,
-      :gamma, :dirichlet_eta].each do |fun|
+      :gamma, :dirichlet_eta
+    ].each do |fun|
 
       context "##{fun}" do
-        let(:class_name) {
+        let(:class_name) do
           f = fun.to_s
           if f.start_with?('a')
-            f.sub(/^../) {|ls| ls.upcase }
+            f.sub(/^../, &:upcase)
           else
             f.capitalize
           end
-        }
+        end
```

Comment:
To be honest, for me the rule "`do..end` for multiline blocks" could be disabled at least for `spec/` folder.